projects/storefrontapp-e2e-cypress/cypress/integration/regression/site-context/currency/currency-checkout-page.e2e-spec.ts

```diff
@@ -14,6 +14,7 @@ describe('Currency switch - checkout page', () => {
       win.localStorage.clear();
     });
     cy.requireLoggedIn();
+    siteContextSelector.doPlaceOrder();
```

Comment:
I think it started here first for placing the order as it always does it, and transitioned to the `it` to try and fix it as well, so I don't think this is a fix for the error

Reply:
if I recall, when it was in this position, the `should have basic data` started failing and it's failing for me locally at times

Reply:
All in all, I'm saying the above due to the failure that I got in headless after 5 runs 

<img width="1440" alt="Screen Shot 2020-04-08 at 12 41 28 AM" src="https://user-images.githubusercontent.com/29099466/78745614-b62dc500-7932-11ea-8ef5-b5d534d9f6d5.png">

Reply:
I left it in the same place, as this is more like test setup, similarly to `requireLoggedIn`. The issue presented above seems to be caused, by something else. Flakiness of tests after last fixes dropped. I got 100% success rate on 20 attempts. The issue above was caused, by language change happening to fast (causing setting address request to cancel). The only thing that I think could fail this test is `Jalo` errors, but didn't notice it yet.